src/ontology/mondo-edit.obo

```diff
@@ -440960,6 +441008,19 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/65
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2614 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700091
+name: disorder caused by ring chromosome
+def: "Chromosomal anomaly consisting of the presence of a ring chromosome. A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome." [https://orcid.org/0000-0002-4142-7153, NCIT:C3360]
+synonym: "r" EXACT [NCIT:C3360]
+synonym: "supernumerary circular chromosome" EXACT [NCIT:C3360]
+xref: MESH:D012303 {source="MONDO:equivalentTo"}
+xref: NCIT:C3360 {source="MONDO:equivalentTo"}
```

Comment:
is this exact? is this in the disorder branch?

I think it's OK to go in as exact for now but we need to return to the issues of mappings to other branches and conflation

Reply:
There are a lot of NCIT and MESH ids that have been used as "disease or disorder" but are actually not a disorder. This is especially an issue in the chromosomal anomaly branch. I have made a note that we should come back to this specific issue.